src/lib/plugin.js

```diff
@@ -159,7 +160,7 @@ module.exports = class PluginVirtual extends EventEmitter2 {
     this._validator.validateOutgoingMessage(message)
     this._safeEmit('outgoing_request', message)
 
-    const response = await this._rpc.call('send_request', this._prefix, [message])
+    const response = await this._rpc.call('send_message', this._prefix, [message])
     this._validator.validateIncomingMessage(response)
```

Comment:
yes, as discussed on slack, I'll change this._validator.validateIncomingMessage so that it accepts `"true"` as a response if the rpc endpoint used was send_message, but still expects a response object if the rpc endpoint used was send_request.